.github/workflows/ci.yml

```diff
@@ -8,7 +8,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2.3.4
-    - uses: ammaraskar/sphinx-action@0.4
       with:
-        docs-folder: "."
-        build-command: make html linkcheck
+        fetch-depth: 0
+    - uses: cachix/install-nix-action@v13
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
+    - name: Setup tmate session
+      uses: mxschmitt/action-tmate@v3
```

Comment:
@garbas won't this block?

Reply:
The problem is that it never gets past this action as it just listens for SSH?

Reply:
@domenkozar I removed the tmate session. Can you approve the build once again?